src/components/ShowCount/styles.scss

```diff
@@ -0,0 +1,18 @@
+.container {
```

Comment:
I think you need to use some specific names for your CSS classes to not overwrite each other.